todo/signals.py

```diff
@@ -0,0 +1,24 @@
+import django.dispatch
+from django.dispatch import receiver
+
+from todo.utils import toggle_task_completed
+
+# ### REGISTER SIGNALS ###
+
+# Demonstrates registering a custom signal
+pizza_done = django.dispatch.Signal(providing_args=["toppings", "size", "task_id"])
+
+
+# ### HANDLE SIGNALS ###
+
+# Demonstrates receiving a custom signal
+# (which in turn calls an existing todo function, but could do anything)
+@receiver(pizza_done)
```

Comment:
These handlers don't need to live in signals.py - you could decorate any function in your project and everything will get wired up when the app starts.